source/read-data-from-mongo.txt

```diff
@@ -18,17 +18,20 @@ Read Data From MongoDB
    :description: Learn how to use the Java Reactive Streams driver to read data from MongoDB.
    :keywords: usage examples, see, crud, access, code example
 
-.. TODO: change toctree links when other pages are merged in.
-.. .. toctree::
-..    :titlesonly:
-..    :maxdepth: 1
-
-..    /read/specify-query
-..    /read/retrieve-data
-..    /read/specify-fields
-..    /read/specify-documents
-..    /read/count-documents
-
+.. toctree::
+    :titlesonly:
+    :maxdepth: 1   
+
+    /read/specify-a-query
+    /read/retrieve-data
+    /read/specify-fields-return
+    /read/specify-documents-to-return
+    /read/count-documents
+    /read/read-preference
+    /read/text-search
+    /read/geo
+    /read/change-stream
+    
 Overview
 --------
 
```

Comment:
S: I can't comment on the lines directly (60-61), but remove the repeated "consume Publisher instances" phrase like you did on the Write Data landing page.